server/subscribe.go

```diff
@@ -16,6 +16,8 @@ import (
 	jira "github.com/andygrunwald/go-jira"
 	"github.com/pkg/errors"
 
+	"github.com/trivago/tgo/tcontainer"
```

Comment:
This package doesn't have a big community, my worry is that It can become deprecated soon-ish in the future. Is the functionality it gives us very valuable? WDYT?

Reply:
I think we actually have to use this because it's the underlying data structure being used by https://github.com/andygrunwald/go-jira for arbitrary field types. I'm not sure if there's a way around this.

Reply:
@javaguirre I think we actually have to use this because it's the underlying data structure being used by https://github.com/andygrunwald/go-jira for arbitrary field types. I'm not sure if there's a way around this.